server/apps/publish/archive_publish.py

```diff
@@ -258,6 +258,8 @@ def _publish_package_items(self, package, updates):
                     package_item = super().find_one(req=None, _id=guid)
 
                 subscribers = self._get_subscribers_for_package_item(package_item)
+                PackageService().update_field_in_package(updates, package_item[config.ID_FIELD],
```

Comment:
Sorry for the late notice. `PackageService` is already initialized in `BasePublishService` with variable name `package_service`. Apart from this it's been initialized in multiple places across the module. Can we replace the initialization with `self.package_service`?